emission/core/wrapper/suggestion_sys.py

```diff
@@ -193,65 +235,90 @@ def return_address_from_location_yelp(location='0,0'):
             business_tuple = check_against_business_location(location, chk)
             if business_tuple[0]: #If true, the lat, lon matches a business location and we return business name
                 address_comp = cleaned[0]['long_name'] + ' ' + cleaned[1]['short_name'] 
-                # print(address_comp)
-                # print(business_tuple[1])
-                # print(cleaned[3]['short_name'])
                 return business_tuple[1], cleaned[3]['short_name'], address_comp
             else: #otherwise, we just return the address
-                # print(cleaned[0]['long_name'])
-                # print(cleaned[1]['short_name'])
-                # print(cleaned[3]['short_name'])
                 return cleaned[0]['long_name'] + ' ' + cleaned[1]['short_name'] + ', ' + cleaned[3]['short_name']
         except:
             raise ValueError("Something went wrong")
 
+'''
+YELP API: Function to find the business matching the address
+'''
 def match_business_address(address):
     business_path = SEARCH_PATH
     url_params = {
         'location': address.replace(' ', '+')
     }
-    return request(API_HOST, business_path, API_KEY, url_params)
+    return request(API_HOST, business_path, YELP_API_KEY, url_params)
+
 '''
-Function to find the review of the original location of the end point of a trip
+YELP API: Function to find the review of the original location of the end point of a trip
 '''
 def review_start_loc(location = '0,0'):
     try:
         #Off at times if the latlons are of a location that takes up a small spot, especially boba shops
 
         #IF RETURN_ADDRESS_FROM_LOCATION HAS A BUSINESS LOCATION ATTACHED TO THE ADDRESS
-        if (len(return_address_from_location_yelp(location)) == 3):
-            business_name, city, address = return_address_from_location_yelp(location)
+        if (len(return_address_from_location_google(location)) == 3):
+            business_name, city, address = return_address_from_location_google(location)
```

Comment:
why is this still calling `return_address_from_location_google`? I thought you were not using the google API?